crates/aptos-api-tester/src/main.rs

```diff
@@ -126,21 +110,9 @@ async fn main() -> Result<()> {
     Logger::builder().level(Level::Info).build();
     let _mp = MetricsPusher::start_for_local_run("api-tester");
 
-    // test flows on testnet
-    let _ = test_flows(
-        NetworkName::Testnet,
-        Client::new(TESTNET_NODE_URL.clone()),
-        FaucetClient::new(TESTNET_FAUCET_URL.clone(), TESTNET_NODE_URL.clone()),
-    )
-    .await;
-
-    // test flows on devnet
-    let _ = test_flows(
-        NetworkName::Devnet,
-        Client::new(DEVNET_NODE_URL.clone()),
-        FaucetClient::new(DEVNET_FAUCET_URL.clone(), DEVNET_NODE_URL.clone()),
-    )
-    .await;
+    // test flows
+    let _ = test_flows(NetworkName::Testnet).await;
+    let _ = test_flows(NetworkName::Devnet).await;
```

Comment:
similarly you could join these together, because right now it runs Testnet, then it runs devnet.  Though separation in this case is probably good